parser/page.go

```diff
@@ -22,13 +23,9 @@ const (
 )
 
 var (
-	delims = [][]byte{
-		[]byte(YAML_DELIM_UNIX),
-		[]byte(YAML_DELIM_DOS),
-		[]byte(TOML_DELIM_UNIX),
-		[]byte(TOML_DELIM_DOS),
-		[]byte(JSON_LEAD),
-	}
+	delims = regexp.MustCompile(
+		"^(" + regexp.QuoteMeta(YAML_DELIM) + "\\s*\n|" + regexp.QuoteMeta(TOML_DELIM) + "\\s*\n|" + regexp.QuoteMeta(JSON_LEAD) + ")",
```

Comment:
It is better to use a raw string literal (string quoted by a back quote) as a regexp string. It allows simply writing `\s*\n|`, no need to escape `\s` by a back slash